src/zilsd.adoc

```diff
@@ -0,0 +1,294 @@
+[[sec:zilsd]]
+== "Zilsd", "Zclsd" Extensions for Load/Store pair for RV32, Version 1.0
+
+The Zilsd & Zclsd extensions provide load/store pair instructions for RV32, reusing the existing RV64 doubleword load/store instruction encodings.
+
+Operands containing `src` for store instructions and `dest` for load instructions are held in aligned `x`-register pairs, i.e., register numbers must be even. Use of misaligned (odd-numbered) registers for these operands is _reserved_.
+
+Regardless of endianness, the lower-numbered register holds the
+low-order bits, and the higher-numbered register holds the high-order
+bits: e.g., bits 31:0 of an operand in Zilsd might be held in register `x14`, with bits 63:32 of that operand held in `x15`.
+
+[[zilsd, Zilsd]]
+=== Load/Store pair instructions (Zilsd)
+
+The Zilsd extension adds the following RV32-only instructions:
+
+[%header,cols="^1,^1,4,8"]
+|===
+|RV32
+|RV64
+|Mnemonic
+|Instruction
+
+|yes
+|no
+|ld rd, offset(rs1)
+|<<#insns-ld>>
+
+|yes
+|no
+|sd rs2, offset(rs1)
+|<<#insns-sd>>
+
+|===
+
+As the access size is 64-bit, accesses are only considered naturally aligned for effective addresses that are a multiple of 8.
+In this case, these instruction are guaranteed to not raise an address-misaligned exception.
```

Comment:
"these instruction" should be "these instructions"

Reply:
will fix this here first: https://github.com/riscv/riscv-zilsd/pull/57

Reply:
fixed in a56eb9de